dashboard/app/controllers/transfers_controller.rb

```diff
@@ -0,0 +1,53 @@
+class TransfersController < ApplicationController
+  before_filter :authenticate_user!
+  # check_authorization
+
+  # TODO: Copying Progress
+  # TODO: some permitting required?
```

Comment:
at least -- you should only be able to transfer students that currently have you as a teacher